src/test/java/com/acrolinx/sidebar/dita/ExternalContentTest.java

```diff
@@ -98,11 +88,10 @@ public void testExternalContentReferencesEmpty()
         final ExternalContent externalContent = new ExternalContentBuilder().build();
         final CheckContent checkContent = new CheckContent("This is a some sample", externalContent);
 
-        assertNotNull(checkContent);
-
-        assertTrue(checkContent.toString().contains("content"));
-        assertTrue(checkContent.toString().contains("externalContent"));
-        assertTrue(checkContent.toString().contains("textReplacements"));
-        assertTrue(checkContent.toString().contains("entities"));
+        String string = checkContent.toString();
```

Comment:
Maybe rename the variable to _content_?

Reply:
The CheckContent.toString() method does not just return "some" String but a JSON formatted String (using the Gson library). IMHO this violates the https://en.wikipedia.org/wiki/Principle_of_least_astonishment and should therefore have done differently, e.g. by adding a separate "getJsonString()" method (or at least add a clear Javadoc comment for the toString() method). Making such a change now would be outside the scope of this PR, maybe we can improve this in the future. For now I pushed another commit where I rename the variable to "jsonString" for added clarity.